Tabloid/client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tabloid-client",
-  "proxy": "https://localhost:5001",
+  "proxy": "https://localhost:44360",
```

Comment:
This is not necessary if you use the project name to run the api in VS rather than IIS Express.

Reply:
You will be changing this setting for your teammates as well if this is pushed.